examples/calculator/calculator.algo.ts

```diff
@@ -33,12 +33,12 @@ class Calculator extends Contract {
    *
    * @returns The result of the operation
    */
-  doMath(a: uint64, b: uint64, operation: string): uint64 {
+  doMath(a: uint64, b: uint64, operation: utf8): uint64 {
     let result: uint64;
 
-    if (operation === 'sum') {
+    if (operation === Bytes`sum`) {
```

Comment:
It would be great to have a separate constructor function for utf-8 strings rather than using `Bytes` otherwise it's a bit confusing (e.g. "what does comparing a string with bytes mean?").

```suggestion
    if (operation === Str`sum`) {
```

Reply:
The answer to "what does comparing a string with bytes mean?" is "the same thing as comparing bytes to bytes". The underlying data is the exact same, `utf8` here is just a signal of what the ABI type should be in the signature, nothing more or less.